docs/monitoring.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: default
+title: Monitoring for Migrations
+nav_order: 13
+permalink: /monitoring
+---
+
+
+# Monitoring for Migrations
```

Comment:
Please add screenshots of the dashboard(s) at relevant places throughout. 
( know its not there due to this being a draft review, just adding it here for completeness)